ios/browser/api/sync/brave_sync_internals.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_IOS_BROWSER_API_SYNC_BRAVE_SYNC_INTERNALS_H_
+#define BRAVE_IOS_BROWSER_API_SYNC_BRAVE_SYNC_INTERNALS_H_
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import <Webkit/Webkit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+OBJC_EXPORT
+@interface BraveSyncInternalsView: UIView
+- (instancetype)init NS_UNAVAILABLE;
+- (void)loadURL:(NSString *)urlString;
```

Comment:
Why have it take a URL when the class is specialized to sync internals? IMO this method should be replaced with something that begins the internals request or removed entirely and loaded in `viewDidLoad`

Reply:
Just realized this is a `UIView` instead of `UIViewController`, maybe it should become a controller instead since we are using it as a standalone screen

Reply:
I think a general purpose Chromium web view to export would be good but will require more work and thought anyways. I am basically saying put that WKWebView into a `UIViewController` (like you would in brave-ios) here instead as there is not really any reason you would use it as a `UIView` in its current state.